server/email/email-router.js

```diff
@@ -0,0 +1,21 @@
+var express = require('express');
+var path = require('path');
+
+module.exports = function(app, io){
+  app.use(express.static(__dirname));
```

Comment:
In an iminent refactor im imagining sending back a different html file if the request failed to go through